Document/Document.ipynb

```diff
@@ -2980,42 +2769,349 @@
     }
    ],
    "source": [
-    "print(cp4)"
+    "print(cp4)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Overall and class benchmark lists are available in `CLASS_BENCHMARK_LIST` and `OVERALL_BENCHMARK_LIST`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 85,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "['AUCI', 'DPI', 'MCCI', 'NLRI', 'PLRI', 'QI']\n",
+      "['SOA1', 'SOA2', 'SOA3', 'SOA4', 'SOA5', 'SOA6']\n"
+     ]
+    }
+   ],
+   "source": [
+    "from pycm import CLASS_BENCHMARK_LIST,OVERALL_BENCHMARK_LIST\n",
+    "print(CLASS_BENCHMARK_LIST)\n",
+    "print(OVERALL_BENCHMARK_LIST)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<ul>\n",
+    "    <li><span style=\"color:red;\">Notice </span> :  `overall_benchmark_weight` and `class_benchmark_weight`, new in <span style=\"color:red;\">version 3.3 </span> </li>\n",
+    "</ul>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### ROC curve"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`ROCCurve`, added in `version 3.7`, is devised to compute the Receiver operating characteristic (ROC) or simply ROC curve. In ROC curves, the Y axis represents the True Positive Rate, and the X axis represents the False Positive Rate. Thus, the ideal point is located at the top left of the curve, and a larger area under the curve represents better performance. ROC curve is a graphical representation of binary classifiers' performance. In PyCM, `ROCCurve` binarizes the output based on the \"One vs. Rest\" strategy to provide an extension of ROC for multi-class classifiers. By getting the actual labels vector and the target probability estimates of the positive classes, this method is able to compute and plot TPR-FPR pairs for different discrimination thresholds and compute the area under the ROC curve. \n",
```

Comment:
`Receiver Operating Characteristic`

Reply:
fixed in https://github.com/sepandhaghighi/pycm/pull/469/commits/3bd4cf43197b3a479d77f89859faf3bf8fdd82eb